docs: separate Debian and Homebrew install instructions for previous DDEV versions, link to it from Building, Testing, and Contributing page (#6631)
ddev/ddev: docs/content/developers/building-contributing.md

Download and unzip the appropriate binary and place it in your `$PATH`.

> ## Install a previous version
> 
> You can also [install an old version of DDEV](https://ddev.readthedocs.io/en/stable/users/usage/faq/#how-can-i-install-a-specific-version-of-ddev).

Reviewer: Thank you, I propose to add two more word for search: rollback and downgrade.

```suggestion
## Rollback to a previous version

You can also [downgrade to a previous version of DDEV](../users/usage/faq.md#how-can-i-install-a-specific-version-of-ddev).
```

Author: Great suggestion, thanks @stasadev!
